client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -331,16 +334,17 @@ const mapStateToProps = ( state, { cardId, purchaseId } ) => ( {
 } );
 
 function useAssignablePaymentMethods() {
+	const translate = useTranslate();
 	const { isStripeLoading, stripeLoadingError, stripeConfiguration, stripe } = useStripe();
 
-	const paypalMethod = useCreatePayPal();
-
 	const stripeMethod = useCreateCreditCard( {
 		isStripeLoading,
 		stripeLoadingError,
 		stripeConfiguration,
 		stripe,
 		shouldUseEbanx: false,
+		shouldShowTaxFields: true,
+		activePayButtonText: translate( 'Save card' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Save card', { context: 'Button label'} )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-cards-back-to-update-payment-method-screen)